db/migration/1661264304751-MigrateSelectedData.ts

```diff
@@ -0,0 +1,118 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import { excludeUndefined, sortBy, uniq } from "../../clientUtils/Util.js"
+
+import { entityNameById } from "./data/entityNameById.js"
```

Comment:
Motivation for this file, mapping 7500 entity ids to their name:

If we were to run this migration _only_ as a database migration then we could, of course, construct this mapping at runtime. But since we also want to see the effect in the SVG tester, where we don't have a database available, including this entity mapping seemed like the most reasonable option.